source/Energinet.DataHub.MarketParticipant.Application/Handlers/Actor/UpdateActorHandler.cs

```diff
@@ -37,6 +37,7 @@ public sealed class UpdateActorHandler : IRequestHandler<UpdateActorCommand>
         private readonly IOverlappingBusinessRolesRuleService _overlappingBusinessRolesRuleService;
         private readonly IAllowedGridAreasRuleService _allowedGridAreasRuleService;
         private readonly IExternalActorIdConfigurationService _externalActorIdConfigurationService;
+        private readonly IUniqeMarketRoleGridAreaService _uniqeMarketRoleGridAreaService;
```

Comment:
Fix spelling mistake

```suggestion
        private readonly IUniqueMarketRoleGridAreaService _uniqueMarketRoleGridAreaService;
```